### PR TITLE
[irods/irods_6363] update quota helptext (main)

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

**Review by alanking — APPROVED** [review]
Looks like it clears it up to me. Will defer to @korydraughn for this one.

**Review by korydraughn — COMMENTED** [review]
The wording is much better, but the `ResourceName-or-'total'` parameter still needs to change. Seeing that in help text is just not common.

What if the arg list became something like ...
```
suq User Target Value
```
My personal preference would be to mirror the man pages and capitalize the args.
```
suq USER TARGET VALUE
``` 
Then you could replace things like `first parameter` with `USER parameter` making it even more clear.

**Comment by trel** [comment]
i like it.

**Comment by trel** [comment]
updated, but didn't go all caps as no other iadmin subcommand uses that formatting.

File: src/iadmin.cpp [hunk 1]
```diff
@@ -2561,35 +2561,35 @@ usage( char *subOpt ) {
...
     };
 
     char *sgqMsgs[] = {
-        " sgq Group ResourceName-or-'total' Value (set group quota)",
+        " sgq Group Target Value (set group quota)",
         " ",
         "Set a storage quota for a group.",
         " ",
-        "The first parameter to this sub-command is the name of the iRODS group.",
+        "The User parameter is the name of the iRODS group.",
```
Comment (korydraughn) [inline src/iadmin.cpp]: User -> Group?

Comment (trel) [reply src/iadmin.cpp]: had already caught it - next commit.

**Review by korydraughn — COMMENTED** [review]
Looks like it's ready for squashing.

**Comment by trel** [comment]
squashed

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by trel** [comment]
#'d

**Pull request merged**